package.json

```diff
@@ -196,8 +197,7 @@
     "ts-node": "^8.4.1",
     "tslint": "^5.8.0",
     "tslint-config-prettier": "^1.6.0",
-    "typescript": "^3.5.3",
-    "vscode": "^1.1.36"
```

Comment:
removed, it causes compile error when used along with `@types/vscode`. The official example also doesn't use this anymore https://github.com/microsoft/vscode-extension-samples/blob/main/statusbar-sample/package.json